spec/components/shared/navbar_component_spec.rb

```diff
@@ -34,6 +32,10 @@
   end
 
   context "when JobseekerAccountsFeature is not enabled" do
```

Comment:
Can we remove this context?

Reply:
And any similar ones

Reply:
Generally we do that once the feature flag is removed